src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.NoArmIntrinsics.xml

```diff
@@ -9,6 +9,9 @@
     <type fullname="System.Runtime.Intrinsics.Arm.Aes">
```

Comment:
I think it's worth splitting this up to to arm and arm64 files so we can disable arm64 intrinsics on arm builds.

Reply:
We don't support intrinsics on arm32 at all at this point. Check out the project file, we only support arm intrinsics on arm64:

https://github.com/dotnet/runtime/blob/598426c8ccf2d1d41038d711f2187899d50ff367/src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems#L14

We can split the file in the future when arm32 support is understood.

Reply:
ok, will split that once we support them